src/navigatorBreadcrumbs.jsx

```diff
@@ -17,12 +17,84 @@
  * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
  */
 import cockpit from "cockpit";
-import React from "react";
+import React, { useState } from "react";
 
-import { Button, Flex, FlexItem, PageBreadcrumb } from "@patternfly/react-core";
-import { ArrowLeftIcon, ArrowRightIcon } from "@patternfly/react-icons";
+import {
+    Button,
+    Dropdown,
+    DropdownItem,
+    DropdownList,
+    Flex,
+    FlexItem,
+    PageBreadcrumb,
+    Icon,
+    MenuToggle,
+} from "@patternfly/react-core";
+import { ArrowLeftIcon, ArrowRightIcon, CheckIcon, EllipsisVIcon } from "@patternfly/react-icons";
 
-export const NavigatorBreadcrumbs = ({ currentDir, path, history, setHistory, historyIndex, setHistoryIndex }) => {
+const _ = cockpit.gettext;
+
+const SettingsDropdown = ({ showHidden, setShowHidden }) => {
+    const [isOpen, setIsOpen] = useState(false);
+    const onToggleClick = () => setIsOpen(!isOpen);
+    const onSelect = (_event, _itemId) => setIsOpen(false);
+
+    const onToggleHidden = () => {
+        setShowHidden(showHidden => {
+            localStorage.setItem("cockpit-navigator.showHiddenFiles", !showHidden ? "true" : "false");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-254-20240207-165317-8e15a54a-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)